### ISSUE TITLE
Test on a single Already Perturbed Image

### ISSUE THREAD

**Issue by imransalam** [kabkabm/defensegan#5]
Is there any way to use the already trained celeba model (trained with train.py) and infer for a single perturbed image and see if the model regenerates the original?

**Comment by bibin-sebastian** [comment]
are you looking for reconstructing one image  ?

**Comment by imransalam** [comment]
@bibin-sebastian Yes, I have already perturbed the image with FGSM, I would want to normalize it with a celeba trained network (trained with train.py).
